### PR TITLE
HPCC-19026 Add hidden message on Sign In for Invalid Credentials

### PR DESCRIPTION
**Pull request by kunalaswani**
Add hidden message on Sign In for Invalid Credentials.

Signed-off-by: Aswani, Kunal <kunal.aswani@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by kunalaswani** [comment]
@miguelvazq  please review

**Comment by kunalaswani** [comment]
@GordonSmith can you review, added functionality to remove cookie to clear hidden message on page after submission of username/password

File: esp/files/Login.html [hunk 1]
```diff
@@ -87,6 +95,7 @@
                     <form method="POST" action="/esp/login">
                         <input type="text" name="username" placeholder="username" class="loginStr"></input>
                         <input type="password" name="password" placeholder="password" class="loginStr"></input>
+                        <p id="hidden_msg" class="loginStr">Invalid username or password, try again.</p>
```
Comment (GordonSmith) [inline esp/files/Login.html]: Translations?  (there is a separate Jira for that I think?)

**Review by GordonSmith — APPROVED** [review]
Looks ok - the cookie stuff is a bit alien to me, so assume its correct,

**Pull request merged**